kill.h

```diff
@@ -8,10 +8,10 @@
 typedef struct {
     /* if the available memory AND swap goes below these percentages,
      * we start killing processes */
-    int mem_term_percent;
-    int mem_kill_percent;
-    int swap_term_percent;
-    int swap_kill_percent;
+    unsigned int mem_term_percent;
+    unsigned int mem_kill_percent;
+    unsigned int swap_term_percent;
+    unsigned int swap_kill_percent;
```

Comment:
I don't know, is using `int` a problem?

Reply:
Not a real problem though. I will revert this.